docs2/site/docs/migrations/migration7.md

```diff
@@ -228,6 +228,17 @@ methods defined on an interface. This functionality is available in C# 8.0 and l
 When building a graph type from an interface, methods are built for all inherited methods
 as well as the specified interface's methods. This is by design.
 
+### 11. Add `ErrorInfoProviderOptions.ExposeExceptionDetailsMode` property
```

Comment:
@Shane32 Please review docs for grammar flaws.

Reply:
I looked through it briefly, but I thought perhaps I would take a pass over the entire document before we release 7.0 -- I was going to do that today (my time), after you merge this PR.  Perhaps you can review my changes tonight (your time) and if it's okay with you, I'll release 7.0 after that (tonight my time).

Mainly I'm thinking of re-ordering migration notes, either to make them flow better, or to draw attention to important notes.  Probably fairly light changes.

Reply:
So I merge this one.